pkg/converters/ingress/ingress.go

```diff
@@ -683,6 +686,18 @@ func (c *converter) syncChangedEndpointCookies() {
 	}
 }
 
+func (c *converter) syncEndpointHashes() {
+	for _, backend := range c.haproxy.Backends().Items() {
+		c.syncBackendEndpointHashes(backend)
+	}
+}
+
+func (c *converter) syncChangedEndpointHashes() {
+	for _, backend := range c.haproxy.Backends().ItemsAdd() {
+		c.syncBackendEndpointHashes(backend)
+	}
+}
```

Comment:
what if these funcs were merged with `syncEndpointCookies()` and `syncChangedEndpointCookies()` (removing the `Cookie` suffix)?

Reply:
Sounds fine :)